src/layman/layer/filesystem/input_file.py

```diff
@@ -143,6 +151,45 @@ def check_raster_main_files(main_filepaths, *, check_crs=True):
                                       'expected': 'All main files with the same CRS.',
                                       'crs': crs_list,
                                       })
+        color_interpretations_list = sorted([fs_gdal.get_color_interpretations(main_filepath) for main_filepath in main_filepaths])
+        unique_color_interpretations_list = remove_duplicities(color_interpretations_list)
```

Comment:
unique_color_interpretations_set = sorted(list({tuple(fs_gdal.get_color_interpretations(main_filepath)) for main_filepath in main_filepaths}))